kyuubi-server/src/main/scala/org/apache/kyuubi/events/KyuubiSessionEvent.scala

```diff
@@ -61,16 +60,14 @@ case class KyuubiSessionEvent(
 
 object KyuubiSessionEvent {
   def apply(session: KyuubiSession): KyuubiSessionEvent = {
-    assert(KyuubiServer.kyuubiServer != null)
-    val serverIP = KyuubiServer.kyuubiServer.frontendServices.head.connectionUrl
     KyuubiSessionEvent(
       session.handle.identifier.toString,
       session.protocol.getValue,
       session.sessionType.toString,
       session.name.getOrElse(""),
       session.user,
       session.ipAddress,
-      serverIP,
+      session.connectionUrl,
```

Comment:
can we keep it in conf map?

Reply:
we shall reserve it to keep compatibility